database_schema.md

```diff
@@ -137,8 +137,39 @@ seat_id: {
 ```
 
 ### Matter
-A matter is specifically a legislative matter. A bill, resolution, initiative, etc. It has a sponser which may be a
-person or a body.
+A matter is specifically a legislative matter. A bill, resolution, initiative, etc.
+```
+matter_id: {
+    name: str
+    matter_type_id: str
+    title: str
+    status: str
+    most_recent_event: {
```

Comment:
Can you add an `next_event` with the same attributes as this one.

Reply:
The interesting thing here is that both `next_event` and `most_recent_event` can be nullable

Reply:
I agree that `next_event` can be nullable. But I'm not sure about `most_recent_event`. My understanding is that a matter only shows up in our db only if it was on the agenda for some event.

Reply:
The pipeline rewrite will have multiple input streams. Yes there will be a pipeline that simply says "if you don't have any future data source then we will handle everything" but I want there to be a pipeline for "gather all available data from external source" which would find us matters that haven't entered committee yet

Reply:
Gotcha. I will add `next_event`